EduNLP/I2V/i2v.py

```diff
@@ -317,6 +324,65 @@ def from_pretrained(cls, name, model_dir=MODEL_DIR, *args, **kwargs):
                    tokenizer_kwargs=tokenizer_kwargs)
 
 
+class QuesNet(I2V):
+    """
+    The model aims to transfer item and tokens to vector with QuesNet.
+
+    Bases
+    -------
+    I2V
+    """
+
+    def infer_vector(self, item, tokenize=True, key=lambda x: x, meta=['know_name'], *args, **kwargs):
+        """ It is a function to switch item to vector. And before using the function, it is nesseary to load model.
+
+        Parameters
+        ----------
+        item : str or dict
+            the item of question
+        tokenize : bool, optional
+            True: tokenize the item
+        key : _type_, optional
+            _description_, by default lambdax:x
+        meta : list, optional
+            meta information, by default ['know_name']
+        args:
+            the parameters passed to t2v
+        kwargs:
+            the parameters passed to t2v
+
+        Returns
+        -------
+        token embeddings
+        question embedding
+        """
+        input = self.tokenize(item, key=key, meta=meta, *args, **kwargs) if tokenize is True else item
+        content = input['content_idx']
+        meta_idx = input['meta_idx']
+        if item['ques_answer'].isalpha() and len(item['ques_answer']) == 1 and len(item['ques_options']) > 0:
```

Comment:
We don't need to process answers and false options to get the vector of questions. 
Check the code in `EduNLP/ModelZoo/QuesNet/QuesNet.py` (line203 and function `forward`). we only use `embs` (contains question content and meta info) to get the question embedding. 
`'ques_answer'` and `'ques_options'` may not exist in `item`, which will cause KeyError here.